app/models/concerns/client_file_base.rb

```diff
@@ -30,6 +27,10 @@ module ClientFileBase
       tagged_with('Client Headshot')
     end
 
+    def tags
+      GrdaWarehouse::AvailableFileTag.where(id: tag_list)
+    end
+
     def file_exists_and_not_too_large
       errors.add :client_file, 'No uploaded file found' if (client_file.byte_size || 0) < 100
       errors.add :client_file, 'File size should be less than 4 MB' if (client_file.byte_size || 0) > 4.megabytes
```

Comment:
These need to be passed as `full_message:`. They're not a valid format for "message" which is meant to be prefixed with the attribute name. It ends up looking like this:
![Screen Shot 2023-03-28 at 9 21 28 AM](https://user-images.githubusercontent.com/5333982/228250324-6f8890b7-332a-4724-833e-5cff26f63ec5.png)

Reply:
Fixed